### PR TITLE
Issue14

### PR DESCRIPTION
**Pull request by Ericrobotic**
See commit log for changes

### CODE REVIEW

File: MeepMeeptest/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (bsneade) [inline MeepMeeptest/.gitignore:1]: It looks like extra stuff unrelated to the ticket was checked in, please remove/undo

File: MeepMeeptest/build.gradle [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    id 'java-library'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+repositories {
+    maven { url = 'https://jitpack.io' }
+    maven { url = 'https://maven.brott.dev/' }
```
Comment (bsneade) [inline MeepMeeptest/build.gradle:11]: It looks like extra stuff unrelated to the ticket was checked in, please remove/undo

File: MeepMeeptest/src/main/java/com/example/meepmeeptest/MeepMeeptest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.example.meepmeeptest;
```
Comment (bsneade) [inline MeepMeeptest/src/main/java/com/example/meepmeeptest/MeepMeeptest.java:1]: It looks like extra stuff unrelated to the ticket was checked in, please remove/undo

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/DriveConstants.java [hunk 4]
```diff
@@ -7,6 +7,8 @@
 /*
```
Comment (bsneade) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/DriveConstants.java:7]: These changes should have been on another branch, but lets go ahead and leave them here for now.  We'll have plenty of more calibration to do.

**Comment by Shadowwalker47** [comment]
Looks good!

**Pull request merged**